### PR TITLE
fix #16243

### LINKED ISSUE #16243: python wheel package is "empty"
## Issue description
```
$ tree /nix/store/lsg86rphp6lcadijmnyfbsl3dqy2ni2d-python2.7-wheel-0.29.0
/nix/store/lsg86rphp6lcadijmnyfbsl3dqy2ni2d-python2.7-wheel-0.29.0
├── lib
│   └── python2.7
│       └── site-packages
└── nix-support
    └── propagated-native-build-inputs
```

### Steps to reproduce

install wheel from pythonpackages in order to get setup.py bdist_wheel

## Technical details
```
$ nixos-version
16.09.git.47e8957 (Flounder)
$ nix-env --version
nix-env (Nix) 1.11.2
$ nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion
"16.03.923.9e959ce"

```

im running a nixos from a git checkout due to testing bug-fixes on gnome
im running a nix-env from the stable channel to work on my projects

### PR DESCRIPTION
**Pull request by domenkozar**
fix #16243

cc @FRidh

### CODE REVIEW

File: pkgs/development/python-modules/generic/default.nix [hunk 1]
```diff
@@ -42,6 +42,9 @@
 # Additional flags to pass to "pip install".
 , installFlags ? []
 
+# Raise an error if two packages are installed with the same name
+, catchConflicts ? true
```
Comment (adnelson) [inline pkgs/development/python-modules/generic/default.nix]: Why should this be optional?

**Comment by FRidh** [comment]
With pip we don't have any conflicts issue. Odd it does happen with wheel.

**Comment by domenkozar** [comment]
@FRidh you're right, we should remove https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/generic/catch_conflicts.py#L12 and do the same for pip and setuptools now.

**Comment by FRidh** [comment]
@domenkozar indeed

File: pkgs/development/python-modules/generic/default.nix (hunk 1 above)
Comment (domenkozar) [inline pkgs/development/python-modules/generic/default.nix]: In this diff I disable it for wheel.

**Comment by FRidh** [comment]
We either need to pass the exceptions to `catch_conflicts.py` or simply add `wheel` there as exception. I say we go for the latter (see #16267).

**Comment by FRidh** [comment]
Fix for wheel is pushed in 9f2699fd303095ca76724c804b0d91c662da11b0.
@domenkozar will add the catchConflicts option separately.

**Comment by domenkozar** [comment]
Done in catchConflicts

**Pull request closed without merging**